src/style.css

```
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  }
+
+
+  .stars {
+  display: flex;
+
+
+  }
+
+
+  .stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+
+
+  }
+
+
+  .stars--1 .stars__star:nth-child(-n+1) {
```

```suggestion
  .stars--1 .stars__star:nth-child(-n+1),
 .stars--2 .stars__star:nth-child(-n+2),
    ... {
```
Consider using commas to group selectors